_posts/2023-10-08-amazon_bedrock_part1.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: postv2
+font: serif
+title: "Amazon Bedrock Anatomy"
+description: "Part1: What's Amazon Bedrock and what it does?"
+date: 2023-10-08
+author: Nadia Reyhani
+tags: [Machine Learning, Amazon Web Services, ML, Generative AI, Amazon Bedrock, AWS]
```

Comment:
be great to get a photo attached so when we post it on socials, that is the default image